README.md

```diff
@@ -48,6 +43,16 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+### Key Features <a name="#future-features"></a>
```

Comment:
- [ ] OPTIONAL
Kindly modify this section title from _Key Features_ to _Future Features_ accordingly.